spec/lib/decision_review/service_spec.rb

```diff
@@ -1,7 +1,9 @@
 # frozen_string_literal: true
 
 require 'rails_helper'
+require 'common/exceptions/forbidden'
```

Comment:
I think we should try to keep the requires in the specs to a minimum.  It looks like `Common::Exceptions::Forbidden` and `DecisionReview::ServiceException` are coming from the `DecisionReview::Service`, so that class should be requiring those.  I don't want to end up in a place where we don't spot a missing `require` because the test included what should have been in the class.

Reply:
blocker?

Reply:
If you're going to make me sound all stern about it, yes.
Pretty easy to change in the github editor though

Reply:
whoops. I asked "blocker?" then immediately decided, meh, I'll just remove it, and removed my question. Sorry about the email notification 👍